core/node/proof_data_handler/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "zksync_proof_data_handler"
+version = "0.1.0"
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+zksync_config.workspace = true
+zksync_dal.workspace = true
+zksync_object_store.workspace = true
+zksync_prover_interface.workspace = true
+zksync_types.workspace = true
+zksync_utils.workspace = true
```

Comment:
Nit: AFAICT, this dependency can be avoided. `zksync_utils` is only used for a single `u256_to_h256` conversion, which could be replaced with `H256::from_low_u64_be(2)` (please test that the produced `H256` are indeed equal).